app/admin/tests.rb

```diff
@@ -1,18 +1,79 @@
 ActiveAdmin.register Test do
 
-  # See permitted parameters documentation:
-  # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
-  #
-  # Uncomment all parameters which should be permitted for assignment
-  #
-  # permit_params :status, :number_of_urls
-  #
-  # or
-  #
-  # permit_params do
-  #   permitted = [:status, :number_of_urls]
-  #   permitted << :other if params[:action] == 'create' && current_user.admin?
-  #   permitted
-  # end
-  
+  actions :index, :show
+  filter :status , :as => :select, :collection => Test::STATUS.invert
+
+  index do
+    id_column
+    column 'Urls' do |test|
+      url_ids = SiteDataInfo.where(test_id: test.id).pluck(:url_id)
+      if url_ids.size > 0
+        link_to 'urls', admin_urls_path('q[id_in]' => url_ids)
+      else
+        div ('no urls found')
+      end
+    end
+    column 'Data Infos' do |test|
+      url_ids = SiteDataInfo.where(test_id: 2).pluck(:url_id).to_s
+      link_to 'data info', admin_site_data_infos_path("q[test_id_equals]" => test.id )
+    end
+    column 'Plugins' do |test|
+      link_to 'plugins', admin_plugins_path('q[first_seen_less_than]' => test.id + 1,'q[last_seen_greater_than]' => test.id - 1, :test_id => test.id)
+    end
+    column 'Themes' do |test|
+      link_to 'themes', admin_themes_path('q[first_seen_less_than]' => test.id + 1,'q[last_seen_greater_than]' => test.id - 1, :test_id => test.id)
+    end
+    column 'JS' do |test|
+      link_to 'js', admin_js_infos_path('q[first_seen_less_than]' => test.id + 1,'q[last_seen_greater_than]' => test.id - 1, :test_id => test.id)
+
+    end
+    column 'Status' do |test|
+      status = test.status 
+      options = Test::STATUS.invert
+      if status == options[:FAILED]
+        div ("FAILED"),style: "color: red"
+      elsif status == options[:INITIALIZED]
+        div ("INITIALIZED"),style: "color: orange"
+      elsif status == options[:COMPLETED]
+        div ("COMPLETED"),style: "color: green"
+      else
+        div ("RUNNING"),style: "color: blue"
```

Comment:
do like this?
case(status)
when options[:FAILED]
 div ("FAILED"),style: "color: red"
 ....................